app/controllers/rubric_items_controller.rb

```
@@ -0,0 +1,125 @@
+      flash[:error] = "Failed to update rubric item assignment"
+    end
+
+    # Redirect with a cache-busting parameter to ensure fresh data is loaded
+    redirect_to view_course_assessment_submission_path(
+      @course,
+      @assessment,
+      @submission,
+      params[:header_position],
+      refresh: Time.now.to_i
+    )
+  end
```

RuboCop (1.73)

[convention] Assignment Branch Condition size for toggle is too high. [<6, 28, 10> 30.33/23]

(Metrics/AbcSize)